package/CHANGELOG

```diff
@@ -53,6 +54,7 @@ Fixes
  * Fix groups.py doctests using sphinx directives (Issue #3925, PR #4374)
 
 Enhancements
+ * Enables parallelization for analysis.gnm.GNMAnalysis (Issue #4672)
```

Comment:
We customarily put new entries first but for anything related to parallelization I propose to add them _below_ the entry for Issue #4158, just so that there is more context.

Reply:
Yep, I can adjust the Changelog to add them below Issue #4158 :)